hicommonwealth/commonwealth
# Token launch CTA elements

## Link to Issue
Closes: https://github.com/hicommonwealth/commonwealth/issues/8704

## Description of Changes
- Add token launch button to content creation sidebar, popover, and mobile drawer.
- Add token launch card on dashboard page for non mobile view only

## "How We Fixed It"
N/A

## Test Plan
1. Enable `FLAG_TOKENIZED_COMMUNITY`
2. Visit the dashboard page, and verify you see the token launch card, click on the launch token button and verify you get redirected to `/createTokenCommunity`
3. Open the content creation sidebar from the + icon in app sidebar, and verify you see the token launch button
4. Visit any community, open the content creation popover and verify you see the token launch button
5. Resize to mobile screen, open the content creation drawer and verify you see the token launch button

## Deployment Plan
N/A

## Other Considerations
N/A
